src/types.ts

```diff
@@ -37,6 +37,11 @@ export type Team = {
   members: User[];
 };
 
+export type TeamList = {
+  count: number | null;
```

Comment:
<img width="320" alt="Screenshot 2021-02-14 at 18 45 47" src="https://user-images.githubusercontent.com/50995397/107881316-e354e780-6ef4-11eb-848f-8bd00d3292ef.png">

it's 100% number. it can not be null. fix

Reply:
done